Update documentation for solarlog (#34136)
home-assistant/home-assistant.github.io: source/_integrations/solarlog.markdown

| power_available       | W      | Available power (equals to power_ac - consumption_ac) | 
| usage                 | %      | Usage (equals to consumption_ac / power_ac) |

## Additional data

{% important %}
The additional data is only accessible if the user's password protection is deactivated. Obviously, deactivating password protection is a security risk and should only be done in specific circumstances. In any event, you do this at your own risk.
{% endimportant %}

You can get additional data from the Solar-Log device. To enable this, select the checkbox for extended data in the integration's system options.

The following additional sensor becomes available:

> | name                  | Unit   | Description   |

Reviewer: ```suggestion
| Name                  | Unit   | Description   |
```

|-----------------------|--------|:-------------------------------------------|
| self_consumption_year | kWh    | Annual self-consumed solar power.          |

> In addition, additional information on devices that are connected to the Solar-Log device become available. The following additional sensors are available (all values are per inverter/device):

Reviewer: ```suggestion
In addition, information from devices connected to the Solar-Log device becomes available. The following additional sensors become available (all values are per inverter/device):
```

> | name                  | Unit   | Description   |

Reviewer: ```suggestion
| Name                  | Unit   | Description   |
```

| power_available       | W      | Available power (equals to power_ac - consumption_ac) | 
| usage                 | %      | Usage (equals to consumption_ac / power_ac) |

## Additional data

{% important %}

> The additional data is only accessible, if password protection for the user is deactivated. Obviously this is a security issue and should only be done in specific circumstanses. In any event, you do this at your own risk.

Reviewer: ```suggestion
The additional data is only accessible if the user's password protection is deactivated. Obviously, deactivating password protection is a security risk and should only be done in specific circumstances. In any event, you do this at your own risk.
```

{% endimportant %}

> Users may choose to get additional data from the solarlog device. To enable, check the box for extended data in the system options of the integration.

Reviewer: ```suggestion
You can get additional data from the Solar-Log device. To enable this, select the checkbox for extended data in the integration's system options.
```

The following additional sensor becomes available:

| name                  | Unit   | Description   |
|-----------------------|--------|:-------------------------------------------|
| self_consumption_year | kWh    | Annual self-consumed solar power.          |

> In addition, you can choose to get additional information on devices that are connected to the Solar-Log device. For this, check the box of the respective device in the configuration dialog of the integration.

Reviewer: ```suggestion
In addition, you can choose to get additional information on devices that are connected to the Solar-Log device. For this, select the checkbox of the respective device in the configuration dialog of the integration.
```
